utils/validations.tsx

```diff
@@ -510,18 +510,18 @@ export const getDualFinanceGovernanceAirdropSchema = ({
       ),
     treasury: yup.object().typeError('Treasury is required'),
     amount: yup
-      .number()
+      .string()
       .typeError('Amount is required')
-      .test('amount', 'amount', async function (val: number) {
+      .test('amount', 'amount', async function (val: string) {
         if (!form.treasury) {
           return this.createError({
             message: `Please select a treasury`,
           })
         }
         const numAtomsInTreasury = new BN(
           form.treasury.extensions.token.account.amount
-        ).toNumber()
-        if (numAtomsInTreasury < val) {
+          )
+          if (numAtomsInTreasury.lt(new BN(val))) {
```

Comment:
if we are using a BN, does this still run into the issue with more than 53 bit numbers?

Reply:
Tested only this number 250000000000000000